bool velocity_on
bool streer_on
int32 velocity
int32 streer_angle
